cockpit.fc

```diff
@@ -8,3 +8,5 @@
 /usr/libexec/cockpit-session	--	gen_context(system_u:object_r:cockpit_session_exec_t,s0)
 
 /var/lib/cockpit(/.*)?      gen_context(system_u:object_r:cockpit_var_lib_t,s0)
+
+/var/run/cockpit-ws(/.*)?   gen_context(system_u:object_r:cockpit_var_run_t,s0)
```

Comment:
yes, but in the policy /var/run is an alias to /run, thus ever spec that applies to /var/run automatically applies to /run as well